cli/assets/templates/zig/src/wasm4.zig

```diff
@@ -121,3 +121,7 @@ pub fn trace(x: []const u8) void {
     traceUtf8(x.ptr, x.len);
 }
 extern fn traceUtf8(strPtr: [*]const u8, strLen: usize) void;
+/// Use with caution, as there's no compile-time type checking.
+/// %d and %f expects 32-bit integers and floats, respectively.
```

Comment:
I thought I commented, but now I don't see it.  Are you certain?  Doesn't it take `f64` not `f32`??? We move the pointer 8 bytes for floats in the runtime code.

Reply:
Oops, you're right!

Reply:
Fixed